index.js

```diff
@@ -15,7 +15,11 @@ var PLUGIN_NAME = 'gulp-sass';
 var gulpSass = function gulpSass(options, sync) {
   return through.obj(function(file, enc, cb) {
     var opts,
+        ip,
```

Comment:
[This is more elegant](https://github.com/xdemocle/gulp-sass/commit/0d7449f3362ade32827334ad3d80dfd48f369928). Going to switch to that